site/en/workflows.md

```diff
@@ -494,9 +494,10 @@ need to segment into lines in an extra step.
     </tr>
     <tr>
       <td>ocrd-sbb-textline-detector</td>
-      <td>&nbsp;</td>
-      <td></td>
-	  <td><code>ocrd-sbb-textline-detector -I OCR-D-DEWARP-PAGE -O OCR-D-SEG-LINE</code></td>
+      <td>{"model": /path/to/model"}</td>
+      <td>Models can be found [here](https://qurator-data.de/sbb_textline_detector/); you need to **pass your individual path to the model**
```

Comment:
you need to **pass your individual path to the model**

->

you need to pass the **local path to the model on your hard drive** ?